core/src/test/java/nu/yona/server/analysis/service/ActivityServiceTests.java

```diff
@@ -341,74 +344,33 @@ public void dayActivityDetailInactivity()
 	}
 
 	@Test
-	public void weekActivityDetailInactivity()
+	public void getUserWeekActivityDetail_noActivityPresent_resultWithInactivity()
 	{
 		WeekActivityDto inactivityWeek = service.getUserWeekActivityDetail(userId, getWeekStartDate(LocalDate.now(userAnonZone)),
 				gamblingGoal.getId());
+
 		assertThat(inactivityWeek.getSpread().size(), equalTo(96));
 		assertThat(inactivityWeek.getStartTime(), equalTo(getWeekStartTime(ZonedDateTime.now(userAnonZone))));
 		assertThat(inactivityWeek.getTimeZoneId(), equalTo(userAnonZone.getId()));
 		assertThat(inactivityWeek.getTotalActivityDurationMinutes().get(), equalTo(0));
 	}
 
 	@Test
-	public void spreadShortDurationInMiddleOfCell()
```

Comment:
That's what I thought. I removed a similar comment before completing the review but forgot to check back on this one.